src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -395,4 +395,212 @@ public void testIsBooleanLiteralType() {
                 .isFalse();
     }
 
+    @Test
+    public void testAreOnSameLine() {
+        final int type = TokenTypes.COMPILATION_UNIT;
+        final DetailAstImpl astForTest = new DetailAstImpl();
+        final DetailAstImpl astForTest2 = new DetailAstImpl();
+        astForTest.setType(type);
+        astForTest2.setType(type);
+        final boolean result = TokenUtil.areOnSameLine(astForTest, astForTest2);
```

Comment:
we should set a line number here instead of token type, because `TokenUtil.areOnSameLine()` works with ast line number.

Reply:
Changes Done